plane/source/docs/starting-up-and-calibrating-arduplane.rst

```diff
@@ -136,19 +138,10 @@ adjust your transmitter trims and repeat the FBWA flight test
 .. image:: ../images/CheckFBWADemands.jpg
     :target: ../_images/CheckFBWADemands.jpg
 
-If they are zero, then you need to adjust the ``AHRS_TRIM_X`` (roll) and
-``AHRS_TRIM_Y`` (pitch) for the difference in angle between the
+If they are zero, then you need to adjust it for the difference in angle between the
```

Comment:
is not clear to me what 'they' is ... nav_roll and nav_pitch ? also maybe change 'adjust it' to 'compensate'

Reply:
fixed